input/fsh/modules/biospecimen.fsh

```diff
@@ -85,15 +85,22 @@ Description: "Concentration of the Aliquot"
 * value[x] only Quantity
 * valueQuantity ^short = "Specify the concentration of the aliquot"
 
-/*Combined Profile*/
 
+/* Invariant to require collection for parent samples*/
+Invariant:   collection-parent
+Description: "If collection.method is not unknown, then there should be no parent sample"
+Expression:  "parent.empty() implies collection.method.exists()"
+Severity:    #error
+
```

Comment:
I think we may need to add another constraint the other direction to actually force this, eg, there could be a collection and a parent right now.